src/main/java/frc/robot/commands/Auto/ScoreHighCubeBalance.java

```diff
@@ -36,18 +37,18 @@ public ScoreHighCubeBalance(DriveSubsystem m_drive, FieldSim m_fieldSim, ArmInOu
         // run instantly
         new InstantCommand(m_LED::autoStart),
 
-        new ManipulatorMidGoal(m_ArmPivot, m_ArmInOut, m_Wrist, m_LED).withTimeout(2),
-        new RunCommand(m_Gripper::GripperFast, m_Gripper).withTimeout(2),
-        new RunCommand(m_Gripper::GripperStop, m_Gripper).withTimeout(1),
+        new ManipulatorMidGoal(m_ArmPivot, m_ArmInOut, m_Wrist, m_LED).withTimeout(1),
```

Comment:
For all autos:

I'm not super confident on approving these times. Have we tested these times all work? It looks like each of the 3 autos for both cone/cube have timings adjusted in this PR that are different from what's on main. I thought the only tested auto timing adjustments we made landed in https://github.com/Team537/RobotCode2023/pull/9?

Reply:
The timings are the same as Score Mid Balance, which worked, so I am pretty sure it should be consistent since the scoring aspect is constant throughout all the cone autos. However, we may need to test the high cube auto timings

Reply:
Ohhhh I think I remember now, it was from the last hour on Saturday those changes were made? They were kept locally on your laptop since then?

Reply:
But yeah, I'm not sure what to do about the high cube auto timings. Maybe something to test briefly in the pit to make sure it at least gets into positions?

Reply:
Sounds Good